bundle/direct/dstate/state.go

```diff
@@ -88,9 +91,11 @@ func (db *DeploymentState) Open(path string) error {
 	if err != nil {
 		if os.IsNotExist(err) {
 			db.Data = Database{
-				Serial:  1,
-				Lineage: uuid.New().String(),
-				State:   make(map[string]ResourceEntry),
+				StateVersion: 1,
```

Comment:
Add CLI version and state for the migrate command as well? cmd/bundle/deployment/migrate.go:190-194

Caught by https://github.com/databricks/cli/pull/4129